File: @LV1/1.4/cetvrta/cetvrta.py
x1, y1 = [int(i) for i in input().split(' ')]
x2, y2 = [int(i) for i in input().split(' ')]
x3, y3 = [int(i) for i in input().split(' ')]

if x1 == x2:
    x = x3
elif x1 == x3:
    x = x2
else:
    x = x1

if y1 == y2:
    y = y3
elif y1 == y3:
    y = y2
else:
    y = y1

print(x, y)

File: LV1/1.4/cetvrta/cetvrta.rb
x1, y1 = gets.split.map(&:to_i)
x2, y2 = gets.split.map(&:to_i)
x3, y3 = gets.split.map(&:to_i)

x = if x1 == x2
      x3
    elsif x1 == x3
      x2
    else
      x1
    end

y = if y1 == y2
      y3
    elsif y1 == y3
      y2
    else
      y1
    end

puts "#{x} #{y}"
